src/lib.rs

```diff
@@ -150,7 +150,8 @@ pub async fn start_server(
         .layer(Extension(reqwest::Client::new()))
         .layer(Extension(storage_client(&options.storage).await?))
         .layer(Extension(transcript))
-        .layer(Extension(options.clone()));
+        .layer(Extension(options.clone()))
+        .layer(DefaultBodyLimit::disable());
```

Comment:
Please also add a `RequestBodyLimitLayer` set to something like 1.5*CONTRIBUTION_SIZE, just disabling the limit creates a security vulnerability.

Reply:
sure, that's reasonable. I'm not sure how much size will be enough for this const. In my local environment, the contribution size was 6.6MB. Is it enough to set the limit to 6.6MB * 1.5 = 9.9MB?
cc: @kevaundray

Reply:
Round it to 10MB for now, if it turns out to be too little we'll increase it again :)

Reply:
Great